commons.py

```diff
@@ -17,16 +17,13 @@ def create_table(con: sqlite3.Connection):
 
 
 def get_random_area_code() -> str:
-    return ''.join([F"{random.randint(0, 9)}" for _ in range(0, 6)])
-
+    return str(random.randint(100000, 999999))
 
 def get_random_age() -> int:
-    return random.choice([5, 10, 15])
-
+    return random.choice(range(5,16,5))
 
 def get_random_active() -> int:
-    return 1 if get_random_bool() else 0
-
+    return random.getrandbits(1)
```

Comment:
So this got faster cos of extra function invocation?

Reply:
Yeah, not sure about the impact of this tbh.